docs/GettingStarted.md

```diff
@@ -946,6 +948,35 @@ alternate_client = Demo::Echo::Service.rpc_stub_class.new(
 
 The integration will ensure that the `configured_interceptor` establishes a unique tracing setup for that client instance.
 
+### http.rb
+
+The http.rb integration will trace any HTTP call using the Http.rb gem.
+
+```ruby
+require 'http'
+require 'ddtrace'
+
+Datadog.configure do |c|
+  c.use :httprb, options
+
+  # optionally, specify a different service name for hostnames matching a regex
+  c.use :httprb, describes: /user-[^.]+\.example\.com/ do |httprb|
+    httprb.service_name = 'user.example.com'
+    httprb.split_by_domain = false # Only necessary if split_by_domain is true by default
+  end
+end
+```
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | ----------- | ------- |
+| `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `false` |
+| `distributed_tracing` | Enables [distributed tracing](#distributed-tracing) | `true` |
+| `service_name` | Service name for `httprb` instrumentation. | `'httprb'` |
+| `split_by_domain` | Uses the request domain as the service name when set to `true`. | `false` |
+| `tracer` | `Datadog::Tracer` used to perform instrumentation. Usually you don't need to set this. | `Datadog.tracer` |
```

Comment:
We've removed the `tracer` option from integrations in #1079, we can remove it here too.

Reply:
ah i see, ty. updated, had to clean up the tests and some of the `pin` code to account for this as well after bringing this up to date with master. I think it's good now tho